proto/quicksilver/airdrop/v1/query.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package quicksilver.airdrop.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "quicksilver/airdrop/v1/params.proto";
+
+option go_package = "github.com/ingenuity-build/quicksilver/x/airdrop/types";
+
+// Query provides defines the gRPC querier service.
+service Query {
```

Comment:
Will need queries defined here for ClaimRecordsByZone with corresponding Request/Response structs.